_rules/first-focusable-elements-are-links-to-content-e53727.md

```diff
@@ -1,159 +1,356 @@
+- has either an [accessible name](#accessible-name) or [accessible description](#accessible-description) that communicates that it links to that specific [section of content](#section-of-content).
+
+**Note**: There is no requirement on how many [focusable](#focusable) elements are part of that [initial segment](#initial-segment), nor any requirement to provide a way to determine (programatically or not) where that [initial segment](#initial-segment) stops. Technique [G124: Adding links at the top of the page to each area of the content](https://www.w3.org/WAI/WCAG21/Techniques/general/G124), and this rule, only require that such an set exists.
+
+**Editorial note**: An attempt to clarify what this initial segment has to be is done in the second expectation: it must contain exactly one link for each section of content in the page. I am still not very happy with the formulation, nor with the order of these two expectations. Any suggestions to improve that are welcome…
+
+## Expectation 2
+
+Each [section of content](#section-of-content) in the [document](#https://dom.spec.whatwg.org/#concept-document) is the target of exactly one link from the set of [focusable](#focusable) elements that passes Expectation 1.
+
+## Assumptions
+
+This rule assumes that any global dismissible information that only appears once per site has already been acknowledged and is not displayed any more.
+
+**Note**: The aim of such links is to be able to skip repeated content (headers, navigation bar, ...) when viewing several pages of the same site. Many sites display a cookies policy banner which might be stealing focus until dismissed (usually be viewing and accepting cookies policy). Since that content is _not_ repeated (is it only shown once for the full site), it is not a problem to have it, and it may appear on any page of the site (depending where the user first comes in).
+
+## Accessibility Support
+
+_There are no major accessibility support issues known for this rule._
+
+**Editorial note**: In its current state, the definition of [section of content](#section-of-content) would include, _e.g._ an advertising sidebar as a specific section of content. However, it is not clear that the lack of link to it would be a breach of G124 or SC 2.4.1… Idea?
+
+## Background
+
+- [G124: Adding links at the top of the page to each area of the content](https://www.w3.org/WAI/WCAG21/Techniques/general/G124)
+
+## Test Cases
+
+### Passed
+
+#### Passed Example 1
+
+A list of links to different sections of content each having a focusable semantic link element and when activated moves the focus to the associated section of content.
+
+```html
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<title></title>
+	</head>
+	<body>
+		<ul>
+			<li><a href="#navigation">Skip to navigation</a></li>
+			<li><a href="#search">Skip to search</a></li>
+			<li><a href="#main">Skip to main content</a></li>
+		</ul>
+		<!-- Repeated Content -->
```

Comment:
We probably don't need the comment here, given that the repeated content is more or less the `nav` and `form` elements. Moreover, the rule does not mention repeated content.

OTOH, we probably need to add to descriptions "Sections of content are identified by headings/semantic elements/…"

Reply:
You are right, I removed the references to repeated content and reworded several of these and included the glossary reference for section-of-content. Let me know if it needs anything further.